app/docker/views/networks/networks.html

```diff
@@ -6,7 +6,14 @@
   </rd-header-title>
   <rd-header-content>Networks</rd-header-content>
 </rd-header>
-
+<information-panel title-text="Offline mode" cant-dismiss="true" ng-if="endpointStatus !== 1">
```

Comment:
since it's the same information-panel. Maybe we can make a component out of this?

Reply:
Agree, something like `information-panel-offline`.